validation/src/sobol_estimator_variance/sensitivity_distribution.py

```diff
@@ -1,604 +1,604 @@
-    )
-
-    sTest_ishigami_martinez = SensitivityConfidenceTest(
-        model_ishigami,
-        distribution_ishigami,
-        MartinezSensitivityAlgorithm,
-        FOexact=sexact,
-        TOexact=stexact,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    # POUTRE
-    model_poutre = ot.SymbolicFunction(
-        ["L", "b", "h", "E", "F"], ["F * L^3 / (48 * E * b * h^3 / 12)"]
-    )
-    model_poutre.setName("poutre")
-    L = ot.LogNormal()
-    L.setParameter(ot.LogNormalMuSigmaOverMu()([5.0, 0.02, 0.0]))
-    b = ot.LogNormal()
-    b.setParameter(ot.LogNormalMuSigmaOverMu()([0.2, 0.05, 0.0]))
-    h = ot.LogNormal()
-    h.setParameter(ot.LogNormalMuSigmaOverMu()([0.4, 0.05, 0.0]))
-    E = ot.LogNormal()
-    E.setParameter(ot.LogNormalMuSigmaOverMu()([3e4, 0.12, 0.0]))
-    F = ot.LogNormal()
-    F.setParameter(ot.LogNormalMuSigmaOverMu()([0.1, 0.20, 0.0]))
-    distribution_poutre = ot.ComposedDistribution([L, b, h, E, F])
-
-    sTest_poutre_saltelli = SensitivityConfidenceTest(
-        model_poutre,
-        distribution_poutre,
-        SaltelliSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_jansen = SensitivityConfidenceTest(
-        model_poutre,
-        distribution_poutre,
-        JansenSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_mauntz = SensitivityConfidenceTest(
-        model_poutre,
-        distribution_poutre,
-        MauntzKucherenkoSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_martinez = SensitivityConfidenceTest(
-        model_poutre,
-        distribution_poutre,
-        MartinezSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    # Aggregated Sobol
-
-    model_aggregated = ot.SymbolicFunction(
-        ["X1", "X2", "X3"],
-        ["2*X1 + X2 - 3*X3 + 0.3*X1*X2", "-5*X1 + 4*X2 - 0.8*X2*X3 + 2*X3"],
-    )
-    model_aggregated.setName("AggregatedSobol")
-    distribution_aggregated = ot.ComposedDistribution([ot.Uniform()] * 3)
-
-    sTest_poutre_saltelli = SensitivityConfidenceTest(
-        model_aggregated,
-        distribution_aggregated,
-        SaltelliSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_jansen = SensitivityConfidenceTest(
-        model_aggregated,
-        distribution_aggregated,
-        JansenSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_mauntz = SensitivityConfidenceTest(
-        model_aggregated,
-        distribution_aggregated,
-        MauntzKucherenkoSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
-
-    sTest_poutre_martinez = SensitivityConfidenceTest(
-        model_aggregated,
-        distribution_aggregated,
-        MartinezSensitivityAlgorithm,
-        savefig=True,
-        plot_figure=True,
-    )
+#! /usr/bin/env python
```

Comment:
Same here. What changed in sensitivity_distribution.py?

Reply:
same here